x-pack/plugins/enterprise_search/common/constants.ts

```diff
@@ -229,39 +233,76 @@ PUT /my-index-000001
   }
 }
 
-# Create an API key
+# 2. Create a connector
+PUT _connector/postgresql-id
+{
+  "name": "My PostgreSQL connector",
+  "index_name":  "connector-postgresql",
```

Comment:
could change `name ` and `index_name` to be more generic so it would work with any service type, that would require less editing in later steps if user wanted to try a different service type

Reply:
I intentionally changed these names to be more prominent that this creation is for a PostgreSQL example. At least to me makes better understand what symbols of the code I should change if I want to go for another type of connector. If we want to go for a generic approach indeed we should change this to `my-index` and `my-connector` i.e

Reply:
yeah exactly understood, just thinking about someone who decides to try this with a different service type, a generic name requires less updates

it's up to you not a strong opinion from me :)

Reply:
Now it's more generic